File: lab/python/matematica_e_operadores/maior_entre_dois_numeros.py
def maior_entre_dois(n1, n2):
    return n1 if n1 > n2 else n2


def maior_entre_dois_enxuta(n1, n2):
    return max(n1, n2)


n1, n2 = map(int, input().split())
print(maior_entre_dois_enxuta(n1, n2))

File: lab/javascript/matematica_e_operadores/maior_entre_dois_numeros.js
function maior_entre_dois(n1, n2){
    return (n1 > n2) ? n1:n2;
}


function maior_entre_dois_enxuta(n1, n2){
    return Math.max(n1, n2);
}


const prompt = require('prompt-sync')();
let [n1, n2] = prompt().split(" ").map(Number);
console.log(maior_entre_dois_enxuta(n1, n2));
